app/models/content_item.rb

```diff
@@ -35,4 +35,8 @@ def to_hash
   def merge(to_merge)
     ContentItem.new(content_item_data.merge(to_merge))
   end
+
+  def phase
+    @content_item_data.fetch("phase", "alpha")
```

Comment:
I wonder if we should crash here if `phase` doesn't exist. It would've caused us to detect the previous bug earlier.

Reply:
@tijmenb Would that mean pages would error if the taxon didn't have the phase? Or doing something like this:

```
  def phase
    @content_item_data.fetch("phase")
  rescue
    nil
  end
```

Reply:
Yes, I think that might be the preferable option. All the taxons should have the phase now, so if they haven't something is wrong and an throwing an error will alert us to it.